docs/en/engines/table-engines/mergetree-family/aggregatingmergetree.md

```diff
@@ -185,7 +166,27 @@ Run the `SELECT` query again, which will return the following output:
 └─────────────────────────┴────────┴───────┘
 ```
 
-Note that the result is the same regardless of which method we used for creating the materialized view.
+In some cases, you might want to avoid pre-aggregating rows at insert time to shift the cost of aggregation from insert time
+to merge time. Ordinarily, it is necessary to include the columns which are not part of the aggregation in the `GROUP BY` 
+clause of the materialized view definition to avoid an error. However, you can make use of the [`initializeAggregation`](/sql-reference/functions/other-functions#initializeaggregation) 
+function with setting `optimize_initialize_aggregation = 0` (it is turned on by default) to achieve this. Use of `GROUP BY` 
```

Comment:
There is no such setting. Maybe you meant `optimize_on_insert`?

Reply:
Yes, sorry 🤦‍♂️ fixed